denorm/denorms.py

```diff
@@ -74,7 +74,7 @@ def get_alldenorms():
     alldenorms = []
     for model in gmodels.get_models(include_auto_created=True):
         if not model._meta.proxy:
-            for field in model._meta.fields + model._meta.local_many_to_many:
+            for field in tuple(model._meta.fields) + tuple(model._meta.local_many_to_many):
```

Comment:
Couple of questions:

1.  `model._meta` is a Django API so I don't see how you even have control over the type here
2.  lists should also sum correctly